lib/oelite/task.py

```diff
@@ -274,6 +277,7 @@ def wait(self, poll=False):
         if self.result is not None:
             # Something bad happened in start
             self.cleanup_context()
+            self.func_end = oelite.util.now()
```

Comment:
Good catch, probably a refactoring artifact. Moreover, there's no reason to store func_end in self, we can just use a local variable (like prefunc_start). Will push a fixed version.